mmv1/products/compute/TargetHttpsProxy.yaml

```diff
@@ -115,7 +124,6 @@ properties:
       resource: 'SslCertificate'
```

Comment:
`ResourceRef` currently means that there's a single target type within the same service- we should revert this to a string. That will probably change the DiffSuppressFunc attached to the schema. We'll want to re-add that DSF by referencing it as a `diff_suppress_func`.

Reply:
I am wondering whether this is neccesary, if I revert it to a string that will mean that our custom expander that depends on it being a `ResourceRef` to pull the `base_url` to create the full resource reference will have to change too.

```
resource_type = property.resource_ref.base_url.split('/').last
tpgresource.ParseGlobalFieldValue("<%= resource_type -%>", <%= var_name -%>, "project", d, config, true)
```